tests/benchmark/app/App.ts

```diff
@@ -0,0 +1,124 @@
+
+import { DNode } from '../../../src/interfaces';
+import { WidgetBase } from '../../../src/WidgetBase';
+import { v, w } from '../../../src/d';
+
+import { Row } from './Row';
+import { Buttons, ButtonConfig } from './Buttons';
+import { Store } from  './Store';
+
+let startTime: number;
+let lastMeasure: string | null;
+
+function startMeasure(name: string) {
```

Comment:
For our internal impl we don’t need to console log the approx run times

Reply:
Removed all code related to console logging